cloudant-client/src/main/java/com/cloudant/client/api/ClientBuilder.java

```diff
@@ -223,6 +260,9 @@ public CloudantClient build() {
         //Build properties and couchdb client
         CouchDbProperties props = new CouchDbProperties(url);
 
+        props.addRequestInterceptors(USER_AGENT_INTERCEPTOR);
```

Comment:
I was thinking there was no point in storing it, but I see your point that if there are multiple clients built we need not have multiple copies.